src/test/java/uk/gov/justice/digital/hmpps/vsip/pages/SelectVisitorsPage.java

```diff
@@ -12,6 +12,10 @@ public void selectVisitorErrorDisplayed() {
 
     public void chooseAPrisoner() { methodsService.click("xpath", "//*[@id='visitor-4729775']"); }
 
+    public void chooseOtherPrisonersVisitor() {
+        methodsService.click("xpath", "//*[@id='visitor-4729773']");
```

Comment:
Is this 4729773 a visitor id, we cant select based on dynamic data,.. need to find a better way to do this!, Anu who used to do this often did this and the tests would break when data changed

Reply:
implemented this